tests/components/minio/test_minio.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for Minio Hass related code"""
+import unittest
+
+from tests.common import get_test_home_assistant
+import homeassistant.components.minio as minio
+
+
+class TestMinioHelper(unittest.TestCase):
```

Comment:
All new tests should be standalone pytest test functions. See the smartthings test suite for example.